lib/src/custom_long_press_gesture_recognizer.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/gestures.dart';
+
+const Duration _kLongPressTimeout = Duration(milliseconds: 250);
+
+const double _kTouchSlop = 5.0;
+
+class CustomLongPressGestureRecognizer extends LongPressGestureRecognizer {
```

Comment:
Let's avoid naming things as "Custom" - that doesn't tell us anything about why we created this, or what it does differently.

Also, always include useful Dart Docs for public classes.